recipe/meta.yaml

```diff
@@ -20,24 +20,24 @@ requirements:
     - python
   run:
     - ansimarkup >=1.0.0
-    - click >=7.0
     - colorama >=0.4
```

Comment:
In cylc-flow this is `0.4.*` so `>=0.4` could cause install issues.

We really don't need to pin colorama to `0.4.*` so it's an easy fix for 8.0a3 but for now I guess we need to specify `colorama>=0.4,<=0.5`?

Reply:
Oh, you are right! Let me fix that.